pybamm/parameters/parameter_values.py

```diff
@@ -152,6 +153,28 @@ def update_from_chemistry(self, chemistry):
         if "sei" in chemistry:
             component_groups += ["sei"]
 
+        if "anode" in chemistry.keys():
+            chemistry["negative electrode"] = chemistry["anode"]
```

Comment:
we could add an error check here that negative electrode and anode aren't both provided

Reply:
Done